internal/cli/cmd/cluster/create.go

```diff
@@ -1213,7 +1216,7 @@ func (f *UpdatableFlags) addFlags(cmd *cobra.Command) {
 	cmd.Flags().StringVar(&f.Tenancy, "tenancy", "SharedNode", "Tenancy options, one of: (SharedNode, DedicatedNode)")
 	cmd.Flags().BoolVar(&f.BackupEnabled, "backup-enabled", false, "Specify whether enabled automated backup")
 	cmd.Flags().StringVar(&f.BackupRetentionPeriod, "backup-retention-period", "1d", "a time string ending with the 'd'|'D'|'h'|'H' character to describe how long the Backup should be retained")
-	cmd.Flags().StringVar(&f.BackupMethod, "backup-method", "snapshot", "the backup method, support: snapshot, backupTool")
+	cmd.Flags().StringVar(&f.BackupMethod, "backup-method", "volume-snapshot", "the backup method, view it by `kbcli cd describe <cluster-definition>`, default is volume-snapshot")
```

Comment:
set a default value "volume-snapshot" is not suitable, the name of backupMethod is specified in backupPolicy, user can also define it in backupPolicy,  "volume-snapshot" is not always a valid name. I suggest default value is "", if it is not set, kbcli can get the backupMethod name which `snapshotVolumes` is true and use it.